torchaudio/models/wav2letter.py

```diff
@@ -1,14 +1,13 @@
-from typing import Optional
-
 from torch import Tensor
 from torch import nn
 
 __all__ = ["Wav2Letter"]
 
 
 class Wav2Letter(nn.Module):
-    r"""Wav2Letter model architecture from the `"Wav2Letter: an End-to-End ConvNet-based Speech Recognition System"
-     <https://arxiv.org/abs/1609.03193>`_ paper.
+    r"""Wav2Letter model architecture from the `Wav2Letter an End-to-End ConvNet-based Speech Recognition System`_.
+
+    .. _Wav2Letter an End-to-End ConvNet-based Speech Recognition System: https://arxiv.org/abs/1609.03193
```

Comment:
nit: How will this be rendered?

Reply:
<img width="831" alt="Screen Shot 2020-07-20 at 17 59 32" src="https://user-images.githubusercontent.com/855818/87990835-cbc69500-cab2-11ea-8af9-eff0a00ddce8.png">